ui-toolkit/modals.md

```diff
@@ -28,7 +28,7 @@ category: UI toolkit
 <div class="content-50 content-first">
   <h4 id="examples">Examples of when to use modals</h4>
   <ul>
-    <li>Confirm an action that can’t be undone, such as “Are you sure you want to delete this?</li>
+    <li>Confirm an action that can’t be undone, such as “Are you sure you want to delete this</li>
```

Comment:
@anselmbradford When you re-add this, please make sure to close the quotes too. Looks like they were missing originally.

Reply:
Heh, how about that. @ielerol want to try out your clone and add the quote?